desktop/src/components/Terminal/Terminal.tsx

```diff
@@ -124,6 +171,36 @@ export const Terminal = forwardRef<TTerminalRef, TTerminalProps>(function T({ fo
           terminalRef.current?.writeln(data)
           termFitRef.current?.fit()
         },
+        highlight(row: number, col: number, len: number, color: string, invertText: boolean) {
```

Comment:
super nit: renaming to `startCol` would make it a bit clearer that we are highlighting by `startColumn + length`. 
Feel free to ignore

Reply:
Done